.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+      - name: Install dependencies
+        run: npm install
+      - name: Build & archive
+        run: |
+          set -x
+          VERSION="${GITHUB_REF##*/}"
+          npm run webpack && tar -zcf "aws-${VERSION}.tar.gz" -C build .
```

Comment:
It only includes (`-C`) the file within the directory to the compressed file (but not the directory itself), which I guess is fine, and may make our future simpler because:
- It prevents "breaking changes" based on the directory name (in the future we can switch `build` with `dist` easier).
- It makes the job of future automations more flexible, cause we won't need to manipulate directories (like renaming it or similar), we'll just need to create the folder where we want to place the files and uncompress (we can always use the `-C` flag again, if needed).

And, I don't expect in the foreseeable future any reason why having a recursive tree within the directory, which is probably the only "downside" of this approach?

Reply:
The `-C` flags what is missing even in the `k6-jslib-summary`; I learned that hardly when I was trying to sketch up a prototype of the semi-automation for the `jslib.k6.io`, so :+1:

Reply:
But I'd like to ask to change the name to the `repo-id`-`version,` e.g., `k6-jslib-aws-v0.0.0.tar.gz`. 

Just the `aws` could mislead that inside something from `aws` or too generic. And using the just repository ID helps us to avoid introducing and maintaining an additional mapping where we need to figure out that the asset of the `k6-js lib-aws` has a name `aws.`

What do you think?

Reply:
> But I'd like to ask to change the name to the `repo-id`-`version,` e.g., `k6-jslib-aws-v0.0.0.tar.gz`.

That's absolutely fine for me! Indeed, I had the doubt about what naming convention to use, and I decided to use just `aws` because that's what's reflected in the [https://github.com/grafana/jslib.k6.io](https://github.com/grafana/jslib.k6.io), but I also agree that generally speaking is probably more clear, less error prone, to use the full name.

Any thoughts other thoughts? @oleiade

Reply:
Hey folks, I'm perfectly happy with those changes 👍🏻 🙇🏻

Reply:
Thanks! Done! cc/ @olegbespalov